src/metabase/actions/settings.clj

```diff
@@ -9,3 +9,11 @@
   :type :boolean
   :visibility :public
   :database-local :only)
+
+(setting/defsetting database-enable-table-editing
+  (i18n/deferred-tru "Whether to enable table data editing for a specific Database.")
+  :default false
+  :type :boolean
+  :visibility :public
+  :database-local :only
+  :export? true)
```

Comment:
Blocking: I don't see anything hiding this setting if you're an OSS user, probably need to move it.